fluent-bit-plugin-loki.yaml

```diff
@@ -0,0 +1,68 @@
+package:
+  name: fluent-bit-plugin-loki
+  version: 3.0.0
+  epoch: 0
+  description: The Fluent Bit loki plugin allows you to send your log or events to a Loki service.
+  copyright:
+    - license: AGPL-3.0-or-later
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - go
+  environment:
+    CGO_ENABLED: "1"
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: b4f7181c7aa9484e66976e8a933111a9b85ea8c2
+      repository: https://github.com/grafana/loki
+      tag: v${{package.version}}
+
+  - uses: go/bump
+    with:
+      deps: github.com/docker/docker@v25.0.5 golang.org/x/net@v0.23.0
+      replaces: go.etcd.io/bbolt=go.etcd.io/bbolt@v1.3.6
+
+  - runs: |
+      make fluent-bit-plugin
```

Comment:
Could we use our go/build melange wrapper here instead? I had a look at the [Makefile](https://github.com/grafana/loki/blob/00bdd2f5b703991b280317ceff0fcf2eed1847d9/Makefile#L508) and it doesn't look like they are doing anything overly special.

go/build has optimisations built in for the binaries, so we should always try and use it where possible

Reply:
Thanks for telling me! I've changed it to use go/build now.